Badge.html

```diff
@@ -22,9 +22,9 @@
                 <li><a href="alert.html">ALERT</a></li>
                 <li><a href="Badge.html">BADGE</a></li>
```

Comment:
Single branch contains multiple components.

Too huge PR, please break it down.

Reply:
Agree. This PR is too big to review.